packages/material-icon/index.js

```diff
@@ -2,34 +2,80 @@ import React from 'react';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 
+import withRipple from '@material/react-ripple';
+
 export default class MaterialIcon extends React.Component {
```

Comment:
i think it is weird to have `export default` but then later in the file have a non-default `export`

Reply:
I need the other export for a test, otherwise its not necessary. I could put it in another file if that's better.